pandas/core/generic.py

```diff
@@ -4685,6 +4685,18 @@ def _where(self, cond, other=np.nan, inplace=False, axis=None, level=None,
             Whether to raise on invalid data types (e.g. trying to where on
             strings)
 
+        Examples
+        ________
+
+        >>> df = pd.DataFrame(np.arange(10).reshape(-1, 2), columns=['A', 'B'])
+        >>> df.where(df % 3 == 0, -df)
```

Comment:
Changed to boolean DataFrame and flipped bools so as to not induce confusion with 0 -> -0 -> 0.